### PR TITLE
Directly include transpiler

### PR DESCRIPTION
**Pull request by AndreyMlashkin**

### CODE REVIEW

File: transpiler/.gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "cmake/modules"]
```
Comment (akokoshn) [inline transpiler/.gitmodules:1]: remove

File: transpiler/.github/workflows/pull_request.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: Handle pull request event
```
Comment (akokoshn) [inline transpiler/.github/workflows/pull_request.yml:1]: in my understanding, should be only one `.gihub/workflow` for all repository.
could you please move logic to the root and check another folders (crypto3, ....)

File: transpiler/.gitmodules (hunk 1 above)
Comment (AndreyMlashkin) [reply transpiler/.gitmodules]: done

File: transpiler/.github/workflows/pull_request.yml (hunk 2 above)
Comment (AndreyMlashkin) [reply transpiler/.github/workflows/pull_request.yml]: removed, logic is already moved

**Review by akokoshn — APPROVED** [review]

**Review by ETatuzova — APPROVED** [review]

**Pull request merged**